src/index.rs

```diff
@@ -157,7 +157,10 @@ impl Indexer {
         doc.add_text(self.fields.id, id);
 
         doc.add_text(self.fields.title, dataset.title);
-        doc.add_text(self.fields.description, dataset.description);
+        doc.add_text(
```

Comment:
This should just not index the document if the information is missing, i.e.

```rust
if let Some(description) = dataset.description {
  doc.add_text(self.fields.description, description);
}
```

Reply:
(Indexing the empty string will have the same effect after tokenization, it just less efficient.)